examples/pubsub_realtime/pubsub_TSN_publisher_multiple_thread.c

```diff
@@ -1414,13 +1416,12 @@ int main(int argc, char **argv) {
     addWriterGroup(server);
     addDataSetWriter(server);
     UA_Server_freezeWriterGroupConfiguration(server, writerGroupIdent);
-    UA_Server_setWriterGroupOperational(server, writerGroupIdent);
```

Comment:
Why is here no enable logic needed? (whole file)

Reply:
I have moved enableWriterGroup call to addWriterGroup() function